Content/Skills/Magic/Nova.cs

```diff
@@ -0,0 +1,74 @@
+﻿using PathOfTerraria.Core.Systems.SkillSystem;
+using Terraria.DataStructures;
+using Terraria.ID;
+
+namespace PathOfTerraria.Content.Skills.Melee;
+
+public class Nova : Skill
+{
+	public override int MaxLevel => 3;
+
+	public override void LevelTo(byte level)
+	{
+		Level = level;
+		Cooldown = MaxCooldown = (15 - Level) * 60;
+		ManaCost = 20 + 5 * Level;
+		Duration = 0;
+	}
+
+	public override void UseSkill(Player player)
+	{
+		if (!CanUseSkill(player))
+		{
+			return;
+		}
+
+		player.statMana -= ManaCost;
+		Timer = Cooldown;
+
+		int damage = (int)(player.HeldItem.damage * (2 + 0.5f * Level));
+		Projectile.NewProjectile(new EntitySource_Misc("NovaSkill"), player.Center, Vector2.Zero, ModContent.ProjectileType<NovaProjectile>(), damage, 2);
```

Comment:
Am I blind? Not seeing this projectile committed

Reply:
Look about 10 lines down lol

Reply:
I usually would, but Nova is almost certainly a one-off so I didn't want to separate into another file. It doesn't even require a sprite (at the moment) so it's just gonna stay in that file.